akka-stream/src/main/boilerplate/akka/stream/scaladsl/GraphApply.scala.template

```diff
@@ -59,26 +55,15 @@ trait GraphApply {
  * INTERNAL API
  */
 object GraphApply {
-  final class GraphImpl[S <: Shape, Mat](override val shape: S, override val module: StreamLayout.Module)
+  final class GraphImpl[S <: Shape, Mat](override val shape: S, override val traversalBuilder: TraversalBuilder)
     extends Graph[S, Mat] {
     
-    override def toString: String = s"Graph($shape, $module)"
+    override def toString: String = s"Graph($shape)"
 
     override def withAttributes(attr: Attributes): Graph[S, Mat] =
-      new GraphImpl(shape, module.withAttributes(attr))
+      new GraphImpl(shape, traversalBuilder.setAttributes(attr))
 
     override def named(name: String): Graph[S, Mat] = addAttributes(Attributes.name(name))
   }
 }
 
-/**
```

Comment:
I am not sure, (I don't know what this was used for), but it will not work with the new layout this way. This can only be fixed I think if I understand what is the use case.